Core/MIPS/RiscV/RiscVCompVec.cpp

```diff
@@ -189,7 +189,39 @@ void RiscVJit::CompIR_VecHoriz(IRInst inst) {
 
 	switch (inst.op) {
 	case IROp::Vec4Dot:
-		CompIR_Generic(inst);
+		// TODO: Maybe some option to call the slow accurate mode?
```

Comment:
I think we can just add a check here for the compat.ini option?

Reply:
Well, that option just does vmmul, I think.  It may be useful at some point to have a slower-but-accurate-vdot mode to check if it's the cause of bugs, without having to go all the way to interpreter for most of VFPU.  But I dunno, maybe it's fine.  Was just thinking about it when I wrote Vec4Dot...

-[Unknown]

Reply:
Hm, I think we use it for a few more ops too? Though yeah, we'd need to track where the Vec4Dot came from, which seems silly. We would rather have two different Vec4Dot opcodes I think, one like this and one more accurate, and emit according to the compat option.